### ISSUE TITLE
Sound Delay

### ISSUE THREAD

**Issue by cjkeating** [miho/SonoAir#18]
Hello.

First of all this is a great piece of software which solves a major flaw with the Sonos system.

One question I have is that when using SonoAir I have a time delay, this doesn't really matter with audio but when watching videos the audio is delayed by 3-5 seconds? Is there anything that I can do to fix this or is it a software issue?

I'm using macOS 10.12.

**Comment by CiscoFish** [comment]
No.  This is the nature of AirPlay.  It maintains a brief buffer i believe.  I don't think there's anything that can be done to change it.

Usually, you would have video and audio sent to something like an apple tv, then it would be in sync.  But in this case, it's going to have that delay no matter what.

**Comment by bartoszkopinski** [comment]
> This is the nature of AirPlay. It maintains a brief buffer

> you would have video and audio sent to something like an apple tv, then it would be in sync

@Ciscog33k you realise Apple TV uses AirPlay, right? According to this, it should be delayed as well (but it isn't).

**Comment by CiscoFish** [comment]
@bartoszkopinski Yep. I have an aTV 2.  That's exactly how the delay is on there too.  I've used other units too, with the same results.

**Comment by kurtlas** [comment]
I fixed that problem and I have to say that ..... its wooorkiiing :)))

Use sonoair in your Mac/Pc. Then use VLC to see your favorite movie. Only thing you have to do, its to delayed the audio to -3000/3500 ms in micronization menu (I thing is under windows tag)

Probably you have to adjust it during a few seconds, but once you have it, then the audio sounds perfectly through your songs system..

I hope it is help.

Regards

**Comment by adrianbj** [comment]
Thanks @kurtlas - that's awesome. In case anyone needs it:

Window > Track Synchronization

![image](https://user-images.githubusercontent.com/871211/68560951-6d5ff780-03f8-11ea-92d5-28d91019ec02.png)
